docs/docs/llms/responses-agent-intro/responses-api-intro.mdx

```diff
@@ -0,0 +1,120 @@
+import { APILink } from "@site/src/components/APILink";
+
+# Responses API Introduction
```

Comment:
Per offline discussion it'd be good to inline this examples into the other page and describe them as ChatAgent vs ResponsesAgent output (maybe under a migration or comparison section?)